tests/resources/queue-event-pass.json

```diff
@@ -29,6 +29,26 @@
+      "messageAttributes": {},
+      "md5OfBody": "9586727cbc9f3312542387099b60982c",
+      "eventSource": "aws:sqs",
+      "eventSourceARN": "arn:aws:sqs:eu-west-2:006106226016:cert-gen-q",
+      "awsRegion": "eu-west-2"
+    },
+    {
+      "messageId": "e48c54a0-7027-4e37-b7e8-c8d231511c60",
+      "receiptHandle": "AQEBJcBvTRZ1W2LSaUJ0g0ELXlqA8WCL4zJxO63wu0YOVhx44xxxPhsnc+/Q9+1vOPYO+3HupEjXzGRSvfPY5rEEJkgCJe4/RQ+q2kU5LsmJEr1qE/CTdIYe5X/75XeMQ523KKpdNsD9tRhyvEpPpSu50byGbz7J0JyR6lu1E6Q4YuB4QNm+ev1obPMLdEt8RhgvIi/NfEfQf0L1r3TPi3wLho1R61PllPm27He8/1CjCnMyWBzgX+DCjJ7vyRXObMZ/MbhMBKbYpeTcejsKpYX//PPr1yvldp1YPC0wPKp+iqmWxoDDeHXbo8xYRFXDA8rnY5RfkwxxffH7o534vYn8FCZEtqybQuo7pumu6Ah9PsC05tP38syU71ltasljGIA35BgCdSO+9r5rTaBnbO9++Q==",
+      "body": "{\"testerStaffId\":\"1\",\"testStartTimestamp\":\"2019-02-26T14:50:44.279Z\",\"odometerReadingUnits\":\"kilometres\",\"testEndTimestamp\":\"2019-02-26T15:02:10.761Z\",\"testStatus\":\"submitted\",\"testTypes\":{\"testNumber\":\"W01A00310\",\"prohibitionIssued\":false,\"testCode\":\"aas\",\"lastUpdatedAt\":\"2019-02-26T15:29:39.537Z\",\"testAnniversaryDate\":\"2019-12-26T15:29:40.032Z\",\"numberOfSeatbeltsFitted\":2,\"testTypeEndTimestamp\":\"2019-02-26T15:02:37.392Z\",\"lastSeatbeltInstallationCheckDate\":\"2019-02-26\",\"createdAt\":\"2019-02-26T15:29:39.537Z\",\"testExpiryDate\":\"2020-02-25T15:29:40.032Z\",\"testTypeId\":\"1\",\"testTypeStartTimestamp\":\"2019-02-26T14:51:09.149Z\",\"certificateNumber\":\"123\",\"seatbeltInstallationCheckDate\":true,\"testTypeName\":\"Paid roadworthiness retest\",\"defects\":[],\"name\":\"Paid retest\",\"testResult\":\"pass\"},\"vehicleClass\":{\"code\":\"s\",\"description\":\"small psv (ie: less than or equal to 22 seats)\"},\"testResultId\":\"1\",\"vehicleSize\":\"small\",\"vin\":\"XMGDE02FS0H012345\",\"testStationName\":\"Abshire-Kub\",\"vehicleId\":\"BQ91YHQ\",\"countryOfRegistration\":\"gb\",\"vehicleType\":\"psv\",\"preparerId\":\"AK4434\",\"preparerName\":\"Durrell Vehicles Limited\",\"odometerReading\":12312,\"vehicleConfiguration\":\"rigid\",\"testStationType\":\"gvts\",\"testerName\":\"CVS, Test, Dev1\",\"vrm\":\"BQ91YHQ\",\"testStationPNumber\":\"09-4129632\",\"numberOfSeats\":50,\"testerEmailAddress\":\"cvs.dev1@dvsagov.onmicrosoft.com\",\"euVehicleCategory\":\"m1\",\"order\":{\"current\":1,\"total\":2}}",
+      "messageAttributes": {},
```

Comment:
Can we have such data in prod for `testerName`? also it seems to be that the payload for testerName is FirstName, MiddleName, LastName